docs/notifications.json

```
@@ -13,5 +8,10 @@
     "id": 82,
     "title": "Upcoming changes to MUI X pricing in 2024",
     "text": "Check out the <a style=\"color: inherit;\" data-ga-event-category=\"Announcement\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-x\" href=\"https://mui.com/blog/mui-x-sep-2024-price-update/\">new pricing updates</a> and how to transition to the new model."
+  },
+  {
+    "id": 83,
+    "title": "Material UI v6 is out!",
+    "text": "This major release includes opt-in CSS extraction, custom properties support, and many more improvements.. Check out the <a style=\"color: inherit;\" data-ga-event-category=\"Announcement\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-x-v7\" href=\"https://mui.com/blog/material-ui-v6-is-out/\">announcement blog post</a>."
```

You need to update the `data-ga-event-label` attribute.

There's a typo:

```suggestion
    "text": "This major release includes opt-in CSS extraction, custom properties support, and many more improvements. Check out the <a style=\"color: inherit;\" data-ga-event-category=\"Announcement\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-x-v7\" href=\"https://mui.com/blog/material-ui-v6-is-out/\">announcement blog post</a>."
```